CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 ### Added
 - Added a new Verifiable Credential v2.0 version of the Utopia Naturalization
   Certificate.
+- Added a new Verifiable Credential v2.0 version of the Utopia Employment
```

Comment:
I added my newer change in #69 in front of the older change on line 7. You added yours below. Should we pick an ordering? I realize this file doesn't care...but the commit log will differ from changelog order...if that matters.

Reply:
Honestly, I put it at the end to not conflict when merging with your change.  It's not a big deal either way.